### PR TITLE
Prime 2: Central Mining Station logic adjustment

### LINKED ISSUE #5912: Dead youtube link in Central Mining Station
In echoes Agon Wastes/Central Mining Station, on the connection Door to Central Station Access -> Door to Command Center Access (top), the highlighted youtube link is dead. 

RDV stable 7.2.0

image

### PR DESCRIPTION
**Pull request by XenoWars**
- Added: Movement (Intermediate) to get from the center platforms in Central Mining Station to the cannons using Space Jump Boots.
- Added: Screw Attack from the cannons to get the item in Central Mining Station before the Pirate fight.
- Fixed: The video link for the Expert Slope Jump in Central Mining Station now links to a working video.

fixes #5912

### CODE REVIEW

File: randovania/games/prime2/logic_database/Agon Wastes.txt [hunk 1]
```diff
@@ -1586,25 +1586,23 @@ Extra - in_dark_aether: False
...
+              Morph Ball Bomb and Space Jump Boots and Bomb Space Jump (Intermediate)
   > Door to Command Center Access (Bottom)
       After Central Mining Station Pirates
   > Event - Central Mining Station Pirates
       Trivial
+  > Center Platforms
+      All of the following:
+          Space Jump Boots and After Central Mining Station Pirates
+          Any of the following:
+              # https://youtu.be/qvaGUSk41g8?list=PLx6vT1x-80zuA1vUepQUSqYOQeHiRNbt5&t=22 Slope Jump
```
Comment (Miepee) [inline randovania/games/prime2/logic_database/Agon Wastes.txt:1601]: 1. please dont reference playlists
2. link ded
3. IMO `Slope Jump: <link>` reads nicer

Comment (XenoWars) [reply randovania/games/prime2/logic_database/Agon Wastes.txt:1601]: i used the videos that were already there.

Comment (Miepee) [reply randovania/games/prime2/logic_database/Agon Wastes.txt:1601]: https://github.com/randovania/randovania/issues/5912 was the issue that started this whole discussion about CMS in the first please, so please do fix it.

Comment (dyceron) [reply randovania/games/prime2/logic_database/Agon Wastes.txt:1601]: Said video is here https://youtu.be/qJQwGw-SuQI

File: randovania/games/prime2/logic_database/Agon Wastes.txt [hunk 2]
```diff
@@ -1586,25 +1586,23 @@ Extra - in_dark_aether: False
...
       After Central Mining Station Pirates
   > Event - Central Mining Station Pirates
       Trivial
+  > Center Platforms
+      All of the following:
+          Space Jump Boots and After Central Mining Station Pirates
+          Any of the following:
+              # Slope Jump: https://www.youtube.com/watch?v=qJQwGw-SuQI
+              Slope Jump (Expert)
+              Scan Visor and Combat/Scan Dash (Expert) and Standable Terrain (Beginner)
```
Comment (Miepee) [inline randovania/games/prime2/logic_database/Agon Wastes.txt:1603]: not an echoes db maintainer, but what i personally like to do for the databases I maintain is to have a `TODO: add video` comment if we don't have a video yet.

**Comment by Miepee** [comment]
changelog conflicts

File: randovania/games/prime2/logic_database/Agon Wastes.txt (hunk 2 above)
Comment (duncathan) [reply randovania/games/prime2/logic_database/Agon Wastes.txt:1603]: ironically this will be counterproductive with #6365 merged

Comment (henriquegemignani) [reply randovania/games/prime2/logic_database/Agon Wastes.txt:1603]: add to the bucket list of new rules to add to that feature

Comment (Miepee) [reply randovania/games/prime2/logic_database/Agon Wastes.txt:1603]: oh right i forgot about that lol

**Comment by XenoWars** [comment]
> changelog conflicts

Was gonna see if we could get that dash video first, but it doesn't quite seem like it. Fine to go ahead without it?

**Comment by Miepee** [comment]
why not just rebase?

**Review by CMos75 — APPROVED** [review]

**Review by dyceron — CHANGES_REQUESTED** [review]
Changelog conflicts

**Review by dyceron — DISMISSED** [review]

**Review by dyceron — APPROVED** [review]

**Pull request merged**